edx_rest_framework_extensions/auth/session/authentication.py

```diff
@@ -1,16 +1,6 @@
-"""
-Deprecated imports for authentication classes that moved.
+""" Session Authentication classes. """
```

Comment:
Since I am working to remove the backward compatibility files and update all the IDA, I decided to move this authentication class as part of this work.
FYI: @ahsan-ul-haq @nasthagiri